source/views/streaming/radio/index.js

```diff
@@ -156,7 +160,18 @@ export default class KSTOView extends React.PureComponent<void, Props, State> {
             {error}
           </View>
 
-          {button}
+          <View style={buttonStyles.buttonWrapper}>
+            {playButton}
+
+            <View style={buttonStyles.space} />
```

Comment:
I was trying to think of a way to avoid wrapping both the `play` and `calendar` (and now `call`) buttons in wrapping views with duplicate `marginRight` stylings. I settled on a simple `<View />` which has a small width. If we try to directly apply the `marginRight` style to the buttons themselves, we end up with the issue we had on Webcams. We do not want to have a larger area be selected because we shifted an element a few pixels.